### DISCUSSION TITLE
Incorrect timezone displayed

### DISCUSSION THREAD

**Discussion by Andre4810** [dbeaver/dbeaver#22015] [DBeaver Development]
Hi. When I create queries and query the DataBase connected to DBeaver, I see an incorrect time.
The real time 11.10 PM (UTC+1, Rome) is displayed as 10.10 PM (UTC).
I tried changing the timezone. The change is saved but I still see the wrong time on every date

<img width="427" alt="Screenshot 2023-11-28 124052" src="https://github.com/dbeaver/dbeaver/assets/152278807/7ee95781-a031-45b5-8252-9351d4271d5f">

<img width="518" alt="Screenshot 2023-11-28 124438" src="https://github.com/dbeaver/dbeaver/assets/152278807/93922de8-f2c9-4dac-be83-c01dec6a2f38">

Can you help me figure out what the error is?
Thanks.

**Comment by LonwoLonwo** [comment]
Please, avoid duplicates https://github.com/orgs/dbeaver/discussions/22016
